hack/ci/e2e-conformance/e2e-conformance_patch.yaml.tpl

```diff
@@ -15,86 +13,6 @@ spec:
         kubeletExtraArgs:
           v: "8"
     verbosity: 8
-    preKubeadmCommands:
```

Comment:
I didn't write this before, so not pay attention on this

why we remove all those code? covered by somewhere?
can you add those info into commit msg?

Reply:
No,the purpose of this commit is removing those code actually. those code patch the manifest  and KUBERNETES_VERSION becomes CI version.
I updated the commit message.